frontend/src/pages/main/home/client/components/graphic/Graphic.tsx

```diff
@@ -0,0 +1,117 @@
+import React, {useEffect, useState} from 'react';
+import {Card, CardContent, Typography} from "@mui/material";
+import "../../../../../../common/styles/container.css";
+import "./styles.css";
+import "../balance/styles.css";
+import {Pie} from "react-chartjs-2";
+import {ArcElement, Chart, Legend, Tooltip} from "chart.js";
+import {Content} from "../../../../../../common/constansts/content";
+import {Transaction, TransactionType} from "../../../../../../types/transactions";
+import {getTransactionForYearService} from "../../../../../../services/transaction";
+import {useAppSelector} from "../../../../../../stores/redux/hooks";
+import {selectLoggedUser} from "../../../../../../stores/redux/loggedUser/loggedUserSlice";
+import {CURRENCY_USD} from "../../../../../../common/constansts/currency";
+import {OTHER_TRANSACTIONS, THRESHOLD_UNIQUE_TRANSACTIONS} from "../../../../../../common/constansts/transactions";
+import {COLORS} from "../../../../../../common/constansts/colors";
+import Finance from "../finance/Finance";
+
+Chart.register(Tooltip, Legend, ArcElement);
+
+const initialTransactions: Transaction[] = [];
+
+const Graphic: React.FC = () => {
+
+    const loggedUser = useAppSelector(selectLoggedUser);
+    const [transactions, setTransactions] = useState<Transaction[]>(initialTransactions);
+    const [income, setIncome] = useState<number>(0);
+    const [expenses, setExpenses] = useState<number>(0);
+
+    const updateTransactions = async () => {
+        const transactionData = await getTransactionForYearService(loggedUser.id, loggedUser.userType, new Date().getFullYear(), new Date().getFullYear());
+        const updatedTransactions = mergeTransactions(transactionData);
```

Comment:
Based on your function name.
```suggestion
        const mergedTransactions = mergeTransactions(transactionData);
```
btw merge is too genereal. It's not clear what is going on.